web-app/src/pages/for-employers/for-employers.tsx

```diff
@@ -236,151 +392,654 @@ function ForEmployers() {
+										to={
+											urlGenerators.queryRoutes
+												.submitSalaryData
+										}
+									>
+										GET STARTED
+									</LinkButton>
+								</BannerButtonContainer>
+							</BannerContent>
+						</Col>
+						<Col xs={12} md={6}>
+							<BannerImage
+								src={bannerImage}
+								alt="employer-banner"
+							></BannerImage>
+						</Col>
+					</Row>
+				</Banner>
 
-				<ProblemPoint>
-					<T.card1part1 /> <Bold>90% - 120%</Bold>
-					<T.card1part2 />{" "}
-					<Bold>
-						<T.card1part3 />
-					</Bold>
-				</ProblemPoint>
-
-				<ProblemPoint>
-					<T.card2part1 />{" "}
-					<Bold>
-						<T.card2part2 />
-					</Bold>
-				</ProblemPoint>
-				<ProblemPoint>
-					<T.card3part1 />{" "}
-					<Bold>
-						<T.card3part2 />
-					</Bold>
-				</ProblemPoint>
-			</SectionContainer>
-
-			<SectionContainer style={{ backgroundColor: "white" }}>
-				<SectionTitle>
-					<T.heading2 />
-				</SectionTitle>
-				<SolutionContainer>
-					<div>
-						<SubsectionTitle>
-							<T.recruitingHeading />
-						</SubsectionTitle>
-						<P>
-							<T.recruitingText />
-						</P>
-					</div>
-					<T.img.exampleJobPost
-						renderer={(t) => (
-							<img
-								className="img-responsive"
-								src={t.src}
-								alt={t.alt}
+				<FeaturesWrapper>
+					<Row>
+						<Col xs={12} md={4}>
+							<CardContent>
+								<CardImageWRapper
+									src={saveMoneyImage}
+									alt="save-money"
+								/>
+								<CardTitle>Save Money</CardTitle>
+								<div>
+									Get two months of free and unlimited job
+									posts by signing up today.You can hire
+									workforce you need easier, faster, and more
+									affordable by reaching over 3000 factory
+									workers on our site.
+								</div>
+							</CardContent>
+						</Col>
+						<Col xs={12} md={4}>
+							<CardContent>
+								<CardIcon>
+									<CardImageWRapper
+										src={saveTimeImage}
+										alt="save-time"
+									/>
+								</CardIcon>
+								<CardTitle>Save Time</CardTitle>
+								<div>
+									We rank and filter all of you application on
+									a weekly basis to find factory workers that
+									are tailored to your needs (availability,
+									skills,education level, and more) so you
+									don't have to start tough CVs yourself
+								</div>
+							</CardContent>
+						</Col>
+						<Col xs={12} md={4}>
+							<CardContent>
+								<CardIcon>
+									<CardImageWRapper
+										src={getResultImage}
+										alt="get-result"
+									/>
+								</CardIcon>
+								<CardTitle>Get Results</CardTitle>
+								<div>
+									You Only pay for results. we charge 5 pesos
+									per job application, rather than charging a
+									monthly subscriotion. This saves you money,
+									gives you a flexibility, to post as many
+									jobs as you need, when you need them, and
+									for however many workers you need.
+								</div>
+							</CardContent>
```

Comment:
Add "Benefit" in front of "<Card..." component names. Example "BenefitCardTitle"